pkg/apis/autoscaling/v1/clusterautoscaler_types.go

```diff
@@ -68,13 +92,22 @@ type ResourceRange struct {
 }
 
 type ScaleDownConfig struct {
+	// Should CA scale down the cluster
 	Enabled bool `json:"enabled"`
+
+	// How long after scale up that scale down evaluation resumes
```

Comment:
Do we want to be more restrictive than a-z? It is supposed to be a duration.

Reply:
It's copy pasted from the upstream documentation.